openapi.yaml

```diff
@@ -97,6 +98,39 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ReferenceDataResponse"
+  /public/v1/sellers/{sellerId}/listings:
+    get:
+      operationId: GetSellersSellerIdListings
+      description: Returns a page of listings for the specified seller ID.
+      tags:
+        - Listing
+      parameters:
+        - $ref: '#/components/parameters/sellerIdPathParam'
+        - $ref: "#/components/parameters/pageQueryParam"
+        - $ref: "#/components/parameters/sizeQueryParam"
+      responses:
+        '200':
+          description: Listings returned.
+          content:
+            application/json:
+              schema:
+                "$ref": "#/components/schemas/SellerListingPage"
+  /public/v1/sellers/{sellerId}/listings/{listingId}:
+    parameters:
+      - $ref: '#/components/parameters/sellerIdPathParam'
+      - $ref: '#/components/parameters/listingIdPathParam'
+    get:
+      operationId: GetSellersSellerIdListingsListingId
+      description: Returns a listing for the specified seller ID and listing ID.
+      tags:
+        - Listing
```

Comment:
We have defined this tag at the beginning of the file, what I am missing? line 18

Reply:
yes, at the beginning we declare all the tags/sections
this tells swagger where to put this one